### PR TITLE
Minor buildsystem fixes

### PR DESCRIPTION
**Pull request by atgeirr**
Resolves issues discovered when building release branch from scratch. Should probably be backported to release branch, but should not affect any binary builds at all, so no need to trigger new build-package-test cycle over this.

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this with downstreams please

**Comment by atgeirr** [comment]
Forgot part of the change. Force-pushed.

**Comment by atgeirr** [comment]
jenkins build this with downstreams please

**Comment by blattms** [comment]
I am interested in what include dirs where missing (which module). Please post or send the module-config.cmake oft that module and ewomes (or at the settings of *_LIBRARIES and *_INCLUDE_DIRS in them)

**Comment by akva2** [comment]
the problem was that the variables was not set. thus the probe in the find module did not set these include paths. They are not affected by the include_directories() call.

**Comment by atgeirr** [comment]
> what include dirs where missing (which module)

All modules downstream of opm-common was affected (would not complete cmake configuration). I am not sure if I understood your request, but the config file of opm-common would have just this:
```
  set (opm-common_INCLUDE_DIRS "/usr/local/include")
```
instead of this:
```
  set (opm-common_INCLUDE_DIRS "/Users/atgeirr/opm/opm-common;/usr/local/include")

```

**Comment by atgeirr** [comment]
If this is ok @blattms would you merge it?

File: cmake/Modules/ewoms-prereqs.cmake [hunk 1]
```diff
@@ -36,10 +36,13 @@ set (ewoms_DEPS
...
 	# valgrind client requests
 	"Valgrind"
 	# quadruple precision floating point calculations
 	"Quadmath"
 	)
+
+if (NOT APPLE)
+	# librt (on some systems necessary for clock_gettime())
+	list(APPEND "librt REQUIRED" ewoms_DEPS)
+endif ()
```
Comment (andlaus) [inline cmake/Modules/ewoms-prereqs.cmake]: I think librt is not used anymore, because the timer code was changed to `std::chrono` for 2017.04.

Comment (atgeirr) [reply cmake/Modules/ewoms-prereqs.cmake]: If so, I guess this could be removed entirely. Could you verify that?

Comment (andlaus) [reply cmake/Modules/ewoms-prereqs.cmake]: I just tested on my Ubuntu 16.04 machine: it works fine without librt.

**Comment by atgeirr** [comment]
Updated to remove the rt lib dep.

**Comment by atgeirr** [comment]
jenkins build this with downstreams please

**Comment by andlaus** [comment]
since this is pretty simple, jenkins is happy and travis only grumbles because of its usual timeout, I take the liberty to merge this now. thanks for testing.

@akva2 will you cherry-pick it to the release branch?

**Pull request merged**